nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -23,6 +23,14 @@
      Support is planned until the end of October 2020, handing over to 20.09.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The graphical installer image starts the graphical session automatically.
+      Before you'd be greeted by a tty and asked to enter <command>systemectl start display-manager</command>.
+      It's still possible to disable the display-manager from running by selecting the <literal>Disable display-manager</literal>
+      quirk in GRUB.
```

Comment:
```suggestion
      quirk in the boot menu.
```

GRUB is an implementation detail; though unlikely some users may not know what a GRUB is.